lib/mix/tasks/oban_web.install.ex

```diff
@@ -0,0 +1,142 @@
+defmodule Mix.Tasks.ObanWeb.Install.Docs do
+  @moduledoc false
+
+  def short_doc do
+    "Installs ObanWeb dashboard into your Phoenix application"
+  end
+
+  def example do
+    "mix oban_web.install"
+  end
+
+  def long_doc do
+    """
+    #{short_doc()}
+
+    This task configures your Phoenix application to use the ObanWeb dashboard:
+
+    * Adds the required router import
+    * Sets up the dashboard route at "/oban"
+      * puts it inside the :dev_routes conditional
+
+    ## Example
+
+    ```bash
+    #{example()}
+    """
+  end
+end
+
+if Code.ensure_loaded?(Igniter) do
+  defmodule Mix.Tasks.ObanWeb.Install do
+    @shortdoc "#{__MODULE__.Docs.short_doc()}"
+
+    @moduledoc __MODULE__.Docs.long_doc()
+    use Igniter.Mix.Task
+
+    @impl Igniter.Mix.Task
+    def info(_argv, _composing_task) do
+      %Igniter.Mix.Task.Info{
+        # Groups allow for overlapping arguments for tasks by the same author
+        # See the generators guide for more.
+        group: :oban,
+        # *other* dependencies to add and call their associated installers, if they exist
+        # i.e `{:foo, "~> 2.0"}`
+        installs: [{:oban, "~> 2.0"}],
+        # An example invocation
+        example: __MODULE__.Docs.example()
```

Comment:
No need for the comments here:

```suggestion
        group: :oban,
        installs: [{:oban, "~> 2.0"}],
        example: __MODULE__.Docs.example()
```